2-mandatory/1-weather-report.js

```diff
@@ -13,6 +13,12 @@
 
 function getTemperatureReport(cities) {
```

Comment:
You are very close to finishing this exercise.

Please think over it and modify your solution in the way that You are going to pass these test cases:
 ✕ should return a temperature report for the user's cities (10 ms)
  ✕ should return a temperature report for the user's cities (alternate input) (1 ms)
  
Tip: Check what exactly you are pushing in line 19 to the array.